app/src/pages/EditCompetition/EditCompetition.jsx

```diff
@@ -18,6 +18,7 @@ import fetchDetailsAction from '../../redux/modules/competitions/actions/fetchDe
 import editAction from '../../redux/modules/competitions/actions/edit';
 import { getCompetition, isEditing, getError } from '../../redux/selectors/competitions';
 import './EditCompetition.scss';
+import RemoveParticipantsModal from '../../modals/RemoveParticipantsModal/RemoveParticipantsModal';
```

Comment:
Can this be moved to the line above it? I like to keep the scss as the last import